src/vma/dev/ring_bond.h

```diff
@@ -172,7 +171,7 @@ class ring_bond_netvsc : public ring_bond
 protected:
 	virtual void slave_create(int if_index);
 
-private:
+public:
```

Comment:
they are used  (for example https://github.com/Mellanox/libvma/pull/607/files#diff-9190366b0d2117456bd385aac29fc2e7R183)

Reply:
The implementation of ring bond::restart is too long. we cant push it like this.
ring bond netvsc inherits from ring_bond, why cant we implement a new virtual restart function ? 
why do we need to have a pointer to the vf_ring and to m_tap_ring? the root cause for the new ring_slave type is to avoid those spespic calls and use virtual function instead. - we should impelemt restart function for each ring_salve.